src/cfn/terminalStackStates.ts

```diff
@@ -1,28 +1,2 @@
-// http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-listing-event-history.html
-// CREATE_COMPLETE | CREATE_FAILED | CREATE_IN_PROGRESS |
-// DELETE_COMPLETE | DELETE_FAILED | DELETE_IN_PROGRESS |
-// DELETE_SKIPPED | UPDATE_COMPLETE | UPDATE_FAILED |
-// UPDATE_IN_PROGRESS.
-
-// StackStatus — values include: "CREATE_IN_PROGRESS",
-// "CREATE_FAILED", "CREATE_COMPLETE", "ROLLBACK_IN_PROGRESS",
-// "ROLLBACK_FAILED", "ROLLBACK_COMPLETE", "DELETE_IN_PROGRESS",
-// "DELETE_FAILED", "DELETE_COMPLETE", "UPDATE_IN_PROGRESS",
-// "UPDATE_COMPLETE_CLEANUP_IN_PROGRESS", "UPDATE_COMPLETE",
-// "UPDATE_ROLLBACK_IN_PROGRESS", "UPDATE_ROLLBACK_FAILED",
-// "UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS",
-// "UPDATE_ROLLBACK_COMPLETE", "REVIEW_IN_PROGRESS"
-
-export default [
-  'CREATE_COMPLETE',
-  'CREATE_FAILED',
-  'ROLLBACK_COMPLETE',
-  'ROLLBACK_FAILED',
-  'DELETE_COMPLETE',
-  'DELETE_FAILED',
-  'UPDATE_COMPLETE',
-  'UPDATE_FAILED',
-  'UPDATE_ROLLBACK_COMPLETE',
-  'UPDATE_ROLLBACK_FAILED',
-  'REVIEW_IN_PROGRESS'
-];
+import {TERMINAL} from './statusTypes';
+export default TERMINAL;
```

Comment:
Is this still useful to have considering it's only exporting an import from somewhere else?

Reply:
I was just being lazy. I'll follow up with a direct import.